packages/react-aria-components/docs/Select.mdx

```diff
@@ -105,6 +105,9 @@ import {Select, SelectValue, Label, Button, Popover, ListBox, Item} from 'react-
     &:disabled {
       border-color: var(--border-color-disabled);
       color: var(--text-color-disabled);
```

Comment:
Should we make the text color more obviously disabled also? I think maybe the SelectValue placeholder selector is overriding this further down.